javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinterTest.java

```diff
@@ -1757,4 +1757,23 @@ void testRemovingInlinedAnnotation_alternate_case() {
 
     }
 
+    @Test
+    void testAtypicalCommentAssignment() {
```

Comment:
I think this test is not very useful because it just prints a class that has not been modified.